postinstall.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const RAW_PACKAGES_URL = 'https://raw.githubusercontent.com/universalbasket/aws-lambda-chrome/master/builds';
+
+const path = require('path');
+const tar = require('tar');
+const { promisify } = require('util');
+const rimraf = promisify(require('rimraf'));
+const mkdir = promisify(require('fs').mkdir);
+const http = require(RAW_PACKAGES_URL.startsWith('https') ? 'https' : 'http');
+
+const chromePath = `/chrome/${process.env.npm_package_archives_chromeFileName}`;
+const nssPath = `/nss/${process.env.npm_package_archives_nssFileName}`;
+const extractPath = path.resolve(__dirname, 'extract');
+
+async function download(path, destination) {
```

Comment:
The `async` here is the only thing incompatible to Node 6, and since it's not required anyway, maybe we can drop it?